.travis.yml

```diff
@@ -6,22 +6,23 @@ compiler:
 os:
     - linux
     - osx
-matrix:
-  exclude:
-    - os: osx
-      compiler: gcc
```

Comment:
to speed the travis check up some.  I didn't see a need for os-x to double check clang/gcc.